src/ar_http_iface_tests.erl

```diff
@@ -1185,6 +1221,75 @@ get_wallet_deposits_test_() ->
 		)
 	end}.
 
+can_post_multiple_txs_per_wallet() ->
+	ar_storage:clear(),
+	{Priv, Pub} = ar_wallet:new(),
+	B0 = ar_weave:init([{ar_wallet:to_address(Pub), ?AR(100), <<>>}]),
+	Node = ar_node:start([], B0),
+	ar_http_iface_server:reregister(Node),
+	Bridge = ar_bridge:start([], Node, ?DEFAULT_HTTP_IFACE_PORT),
+	ar_http_iface_server:reregister(http_bridge_node, Bridge),
+	ar_node:add_peers(Node, Bridge),
+	%% Create and post two transaction from the same wallet
+	TX = (ar_tx:new())#tx{ owner = Pub, reward = ?AR(1) },
+	SignedTX = ar_tx:sign(TX, {Priv, Pub}),
+	TX2 = (ar_tx:new())#tx{ owner = Pub, reward = ?AR(1), last_tx = SignedTX#tx.id },
+	SignedTX2 = ar_tx:sign(TX2, {Priv, Pub}),
+	{ok, {{<<"200">>, _}, _, <<"OK">>, _, _}} =
+		ar_httpc:request(
+			<<"POST">>,
+			{127, 0, 0, 1, 1984},
+			"/tx",
+			[],
+			ar_serialize:jsonify(ar_serialize:tx_to_json_struct(SignedTX))
+		),
+	receive after 100 -> ok end,
+	{ok, {{<<"200">>, _}, _, <<"OK">>, _, _}} =
+		ar_httpc:request(
+			<<"POST">>,
+			{127, 0, 0, 1, 1984},
+			"/tx",
+			[],
+			ar_serialize:jsonify(ar_serialize:tx_to_json_struct(SignedTX2))
+		),
+	receive after 1000 -> ok end,
+	ar_node:mine(Node),
+	receive after 1000 -> ok end,
+	ReadTX = ar_storage:read_tx(SignedTX#tx.id),
+	?assertEqual(<<>>, ReadTX#tx.last_tx),
+	ReadTX2 = ar_storage:read_tx(SignedTX2#tx.id),
+	?assertEqual(SignedTX#tx.id, ReadTX2#tx.last_tx).
```

Comment:
Can we somehow more explicitly assert that the two transactions are in the block?